jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/HoverTests.java

```diff
@@ -17,70 +17,116 @@
 import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
 import static com.jdiai.tools.Timer.waitCondition;
 import static io.github.com.StaticSite.hoverPage;
-import static io.github.com.pages.HoverPage.cartButton;
 import static io.github.com.pages.HoverPage.closeDelayHover;
-import static io.github.com.pages.HoverPage.cookingHeader;
+import static io.github.com.pages.HoverPage.cookingHoverCard;
 import static io.github.com.pages.HoverPage.disabledHover;
 import static io.github.com.pages.HoverPage.musicHoverCards;
 import static io.github.com.pages.HoverPage.openDelayHover;
-import static io.github.com.pages.HoverPage.priceHover;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.not;
 
 public class HoverTests extends TestsInit {
+
+    private static final List<String> SUBHEADINGS = Arrays.asList("New Releases", "Rock", "Mellow Moods");
+    private static final String COOKING_CARD_PRICE = "$14.99";
+    private static final String COOKING_CARD_HEADER_TEXT = "cooking utensils";
+
     @BeforeClass
     public void before() {
         hoverPage.open();
         waitCondition(() -> hoverPage.isOpened());
         hoverPage.checkOpened();
     }
 
-    @Test
-    public void hoverEffectsTest() {
+    @Test(description = "Assert that hover effect is disabled and absent even if card is being hovered")
+    public void hoverEffectIsAbsentTest() {
         disabledHover.show();
-        disabledHover.suppress();
-
-        Stream.of(openDelayHover, closeDelayHover).forEach(hover -> {
-            hover.has().cssClass("elevation-2");
-            hover.show();
-            hover.has().cssClass("elevation-16");
-            hover.has().cssClass("on-hover");
-            hover.suppress();
-            hover.has().cssClass("elevation-2");
+        disabledHover.assertThat().attr("class", not(containsString("on-hover")));
+    }
+
+    @Test(description = "Assert that hover effect is enabled and present only when card is being hovered")
+    public void hoverEffectIsPresentTest() {
+        Stream.of(openDelayHover, closeDelayHover).forEach(item -> {
+            item.has().cssClass("elevation-2");
+            item.show();
+            item.has().cssClass("elevation-16");
+            item.has().cssClass("on-hover");
+            item.suppress();
+            item.has().cssClass("elevation-2");
+        });
```

Comment:
Please, add check 'noCssClass' or assertThat().attr("class", not(containsString("on-hover")))
It's better than 'elevation' feature :)